test/allowlist_for_publicAPI.json

```diff
@@ -2092,6 +2092,7 @@
     "Node",
     "NodeList",
     "NodeSet",
+    "Optional",
```

Comment:
We do not allow new entries in this file. Would you please be able to cleanup the public API for this file such that nothing is confused as public API.

In this case, your simplest way forward is most likely to define `__all__` there to mark what is actually public there.

Reply:
I read the definition of what public api is and think it is suitable to mark Optional as unpublic due to it is the built-in type of python. See this line https://github.com/pytorch/pytorch/pull/135317/files#diff-9d94acf13fbaa041fa5d29da1f79d42d2a361f389706bcc6da67c30c6ec92eacR4